src/start-standalone.js

```diff
@@ -0,0 +1,8 @@
+const uppy = require('./pluggable')
+const app = require('./standalone')
+const PORT = 3020
```

Comment:
Not related to this PR, but it would be nice in the future to have something like `process.env.UPPYSERVER_PORT || 3020` for folks that have requirements around that in their infra